### PR TITLE
refactor: improve formly/form css styling

### PR DESCRIPTION
**Pull request by SGrueber**
## PR Type

[x] Refactoring (no functional changes, no API changes)

## What Is the Current Behavior?
There are some duplicate css stylings for formly and regular form artifacts.

## What Is the New Behavior?
There are no duplicate form stylings any more.

## Does this PR Introduce a Breaking Change?

[x] No

## Other Information

[AB#65786](https://dev.azure.com/intershop-com/cefd1005-00a7-4c79-927f-a16947d1b2e6/_workitems/edit/65786)

### CODE REVIEW

**Review by MaxKless — CHANGES_REQUESTED** [review]
Found a couple of things: 
- The checkmark styling is not green anymore :(
image
- Currently, descriptions on invalid fields are not red. With your PR, they are also styled red. What's the correct behavior here?
image
image

File: src/styles/main.scss [hunk 1]
```diff
@@ -43,7 +43,6 @@
 @import 'global/forms/labels';
 @import 'global/forms/fields';
 @import 'global/forms/forms';
-@import 'global/forms/formly';
```
Comment (shauke) [inline src/styles/main.scss:46]: If you remove the import of `formly.scss` shouldn't the file `formly.scss` be removed as well? Currently it contains unimported styles.

File: src/styles/global/forms/forms.scss [hunk 2]
```diff
@@ -154,3 +160,20 @@ input.form-check-input {
...
+    right: 40px;
+  }
+}
+.validation-offset-0 {
+  .form-control-feedback {
+    right: 10px;
+  }
+}
+
+.formly-empty-label {
```
Comment (shauke) [inline src/styles/global/forms/forms.scss:175]: This seems to be the only CSS class with `formly` in its name. Maybe this is something that could be named more general? Especially in `forms.scss`. ;)

**Review by shauke — CHANGES_REQUESTED** [review]

**Review by MaxKless — APPROVED** [review]
Don't see anything wrong with the code and all my issues work now :)

**Review by shauke — APPROVED** [review]

**Pull request merged**